add community section (#2081)
FlowFuse/website: src/handbook/community/event-and-webinars.md

---
navTitle: Events and Webinars
---

## FlowFuse Webinars and Events

> At FlowFuse, we are committed to delivering value to our customers and users. That's why we conduct monthly webinars and other initiatives to enhance experience and knowledge.

Reviewer: ```suggestion
At FlowFuse, we are committed to delivering value to our customers and users. To enhance their experience and knowledge, we conduct monthly webinars and other initiatives.

```
